src/app/containers/FrontPage/index.stories.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import fixtureData from '../../../../data/prod/pidgin/frontpage';
+import FrontPage from '.';
+
+storiesOf('Front Page', module).add('default', () => {
+  const data = {
+    pageData: fixtureData,
+    status: 200,
+  };
+
+  return (
+    <FrontPage
+      data={data}
+      service="news"
```

Comment:
As the fixture data is `pidgin` maybe this should also be pidgin?

Reply:
If I put `pidgin` as a `service` it gives an error in the `CanonicalConsentBannerContainer` file because is trying to access to the `translations` object which is not in the `lib/config/services/igbo.js` file.

Reply:
maybe a quick comment on this line with a TODO to update the service name once the config is available. Just so others are a ware of the reasoning behind this.